protocol/0013-ACCT-accounts.md

```diff
@@ -146,3 +173,13 @@ One key difference with staking accounts is that the collateral is not held in a
   - The balance can only be delegated to Validators (<a name="0013-ACCT-015" href="#0013-ACCT-015">0013-ACCT-015</a>)
   - The balance cannot be traded, or used as margin, or transferred, or withdrawn (<a name="0013-ACCT-016" href="#0013-ACCT-016">0013-ACCT-016</a>)
   - Delegated stake remains in the trader's staking account (<a name="0013-ACCT-017" href="#0013-ACCT-017">0013-ACCT-017</a>)
+
+### Network treasury
+
+- It is possible to deposit funds from Ethereum directly into the network treasury account by specifying the `0` address and appropriate account type. (<a name="0013-ACCT-026" href="#0013-ACCT-026">0013-ACCT-026</a>)
```

Comment:
I am not sure it's true that you can specify an account type for a transfer from Ethereum? I assume it just always goes to one of the types… cc @ze97286 ?

Reply:
It is the global reward account that deposits made to `0` account get credited to:
https://github.com/vegaprotocol/vega/blob/develop/core/collateral/engine.go#L3704

I've modified the spec to reflect that.